### DISCUSSION TITLE
.NET samples for HttpPlugin

### DISCUSSION THREAD

**Discussion by efonsecab** [microsoft/semantic-kernel#5665] [Q&A] [answered]
Hello, I'm looking for .NET examples on how to use HttpPlugin.

**Comment by sophialagerkranspandey** [comment, marked as answer]
What we've found, is that the model performs better if you describe your APIs as a native function (or as an OpenAPI manifest import). As we approach Build, we are revisiting our plugins and will likely remove plugins like this one that are not as performant or use outdated best practices.
[semantic-kernel/dotnet/src/Plugins/Plugins.Core/HttpPlugin.cs at main · microsoft/semantic-kernel (github.com)](https://github.com/microsoft/semantic-kernel/blob/main/dotnet/src/Plugins/Plugins.Core/HttpPlugin.cs)
